js/modules/Search/Table.js

```diff
@@ -199,6 +199,7 @@ window.GLPI.Search.Table = class Table extends GenericView {
                 this.getElement().trigger('search_refresh', [this.getElement()]);
                 this.hideLoadingSpinner();
                 this.shiftSelectAllCheckbox();
+                this.reloadJS('rateit');
```

Comment:
It looks like this can be simplified to just:
ajax_container.find('.rateit').rateit();

No need to reload the rateit library as it is available when the page loads. The issue is that the library doesn't re-initialize the components automatically after the page is done loading so anything added by AJAX requests needs manually initialized.